src/main/java/org/pac4j/vertx/handler/impl/RequiresAuthenticationHandler.java

```diff
@@ -138,6 +145,33 @@ public void handle(RoutingContext routingContext) {
 
     }
 
+    @Override
+    protected void authorise(final User user, final RoutingContext context) {
+        if (! (user instanceof Pac4jUser)) {
+            throw new TechnicalException("Wrong user type in authorise");
+        }
+
+        Handler<AsyncResult<Boolean>> authHandler = res -> {
+            if (res.succeeded()) {
+                if (res.result()) {
+                    context.next();
+                } else {
+                    context.fail(403);
+                }
+            } else {
+                context.fail(res.cause());
```

Comment:
Again I could extract that for customization if needed. I wasn't sure if you wanted more than the status code of the response, that's all, and your question was obviously clear enough as my answer covered what you were looking for :)